app/views/overrides/activation_keys/_host_media_type_select.html.erb

```diff
@@ -0,0 +1,22 @@
+<% using_hostgroups_page = @host.nil? %>
+<% media_selection = using_hostgroups_page ? use_install_media(@hostgroup) : use_install_media(@host, :selected_host_group => @hostgroup)
+   install_media_radio  = media_selection ? 'checked="checked"':''
+   synced_content_radio  = media_selection ? '' : 'checked="checked"'
+%>
+
+<div class="clear-fix" id="media_selection_section">
+  <div class="form-group ">
+     <label class="col-md-2 control-label" for="use_install_media">Media Selection</label>
+     <div class="col-md-4">
+        <label class="radio-inline">
+          <input data-media-selector="true" id="host_use_install_media_true" name="media_selector" value="install_media" type="radio" <%= install_media_radio  %> /> <%= _('Install Media') %>
```

Comment:
Hm, could we all this "All Media" or something, and put it after Synced Content? It's nitpicky but seems clearer to me.

```
[X] Synced Content [ ] All Media
```

Synced content is also technically install media.